File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.kt
class Solution {
    fun solution(price: Int): Int {
        var answer = price
        if(price in 100000 until 300000){
            answer = (price * 0.95).toInt()
        }else if(price in 300000 until 500000){
            answer = (price * 0.9).toInt()
        }else if(price >= 500000){
            answer = (price * 0.8).toInt()
        }
        return answer
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if price >= 500000:
        answer = price * 0.80
    elif price >= 300000 and price < 500000:
        answer = price * 0.90
    elif price >= 100000 and price < 300000:
        answer = price * 0.95
    else:
        answer = price
    return int(answer)